src/hyperion/parameters/rotation.py

```diff
@@ -104,3 +116,52 @@ def scan_points(self) -> AxesPoints:
     @property
     def num_images(self) -> int:
         return int(self.scan_width_deg / self.rotation_increment_deg)
+
+
+class MultiRotationScan(RotationScanGeneric, SplitScan):
```

Comment:
You can update the hierarchy in `param_hierarchy.puml` and regenerate the diagram.